test/simplifyExpression/breakUpNumeratorSearch/breakUpNumeratorSearch.test.js

```diff
@@ -11,6 +11,7 @@ describe('breakUpNumerator', function() {
     ['(x+3+y)/3', '(x / 3 + 3/3 + y / 3)'],
     ['(2+x)/4', '(2/4 + x / 4)'],
     ['2(x+3)/3', '2 * (x / 3 + 3/3)'],
+    ['(2x + 3)/(2x + 2)', '(1 / (2x + 2) + (2x + 2) / (2x + 2))'],
```

Comment:
same here - try to think of edge cases. I can help you brainstorm if you dunno what to add